make/modules/java.desktop/lib/ClientLibraries.gmk

```diff
@@ -23,16 +23,11 @@
 # questions.
 #
 
-include CopyFiles.gmk
-
-LIBAWT_DEFAULT_HEADER_DIRS := \
-    common/awt/utility \
-    libawt/awt/image \
-    libawt/awt/image/cvutils \
-    libawt/java2d \
-    libawt/java2d/loops \
-    libawt/java2d/pipe \
-    #
+################################################################################
```

Comment:
This file needs an IBM copyright notice.

Reply:
Added